neo/io/asciisignalio.py

```diff
@@ -196,7 +196,13 @@ def read_segment(self, lazy=False):
                                 usecols=self.usecols,
                                 skip_header=self.skiprows,
                                 dtype='f',
+                                filling_values=None,
+                                comments='""',
+                                names=None,
+                                loose=True,
                                 invalid_raise=False)
+
+            nan = np.isnan(sig[:])
```

Comment:
This variable is not used later on. Is there more code missing here?

Reply:
I fixed it.